server/php-queries.h

```diff
@@ -18,51 +19,46 @@ struct FinishedJob;
 struct JobSharedMessage;
 } // namespace job_workers
 
-enum class net_event_type_t {
-  rpc_answer,
-  rpc_error,
-  job_worker_answer,
+namespace net_events_data {
+
+struct rpc_answer {
+  int result_len;
```

Comment:
Please add member initializers here and below.

Reply:
done